sdk/webpubsub/azure-webpubsub/azure/webpubsub/_protocol/_preparers_py3.py

```diff
@@ -0,0 +1,616 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+from typing import Any, IO, List, Optional, Union
+
+from azure.core.pipeline.transport._base import _format_url_section
+from azure.core.protocol import HttpRequest
+from msrest import Serializer
+
+_SERIALIZER = Serializer()
+
+
+def prepare_healthapi_get_health_status(
+    *,
+    api_version: Optional[str] = "2020-10-01",
+    **kwargs: Any
+) -> HttpRequest:
+
+    # Construct URL
+    url = kwargs.pop("template_url", '/api/health')
+
+    # Construct parameters
+    query_parameters = {}  # type: Dict[str, Any]
+    if api_version is not None:
+        query_parameters['api-version'] = _SERIALIZER.query("api_version", api_version, 'str')
+
+    return HttpRequest(
+        method="HEAD",
+        url=url,
+        params=query_parameters,
+    )
+
+
+def prepare_webpubsub_send_to_all(
+    hub: str,
+    *,
+    body: Union[IO, str, IO],
+    excluded: Optional[List[str]] = None,
+    api_version: Optional[str] = "2020-10-01",
+    **kwargs: Any
+) -> HttpRequest:
+    content_type = kwargs.pop("content_type", "application/octet-stream")
```

Comment:
@johanste ideal is not really relying on `content_type` in preparer. But instead, having `data` kwarg accept really any kind of body, and then sniffing the type of `data` to determine `content_type` (if `content_type` is not explicitly provided)